storage-bigtable/src/lib.rs

```diff
@@ -440,8 +435,25 @@ impl LedgerStorage {
                     row_key
                 ))
             })?;
-            let mut cell_data: Vec<TransactionByAddrInfo> =
-                bigtable::deserialize_bincode_cell_data(&data, "tx-by-addr", row_key)?;
+
+            let deserialized_cell_data = bigtable::deserialize_protobuf_or_bincode_cell_data::<
+                Vec<TransactionByAddrInfo>,
```

Comment:
Unfortunately, I think you'll need a separate struct here (akin to `StoredConfirmedBlock`).
The old bincode data won't deserialize into your new `TransactionByAddrInfo` as-is, since there are no bytes for the `block_time` field. (And adding `#[serde(deserialize_with = "default_on_eof")]` to that field won't work, since this is a Vec...)

Reply:
Oh you're right...

Reply:
Oh, that makes sense! I introduced a LegacyTransactionByAddrInfo.

I'll message you to make sure I'm testing this bigtable deserializiation correctly from the running the validator.